### ISSUE TITLE
Improve project saving/loading to the .mantid file - Fitting

### ISSUE THREAD

**Issue by NickDraper** [mantidproject/mantid#3383]
This issue was originally [TRAC 2536](http://trac.mantidproject.org/mantid/ticket/2536)

This ticket is blocked by : 
- [X] [TRAC9970](http://trac.mantidproject.org/mantid/ticket/9970)

Save should store the structure and values of the fit function toolbar.

Load should restore them.

**Comment by samueljackson92** [comment]
I think this is a duplicate of #17445. We'll use the other issue and close this one.
